scripts/init.sh

```diff
@@ -51,20 +52,19 @@ start-parachain)
   fi
 
   ./scripts/run_collator.sh \
-    --chain="${parachain}" --alice \
+    --chain="${parachain}" \
+    --alice \
     --parachain-id="${para_id}" \
     --base-path=$parachain_dir/data \
     --wasm-execution=compiled \
     --execution=wasm \
     --port $((30355 + $para_id)) \
     --rpc-port $((9936 + $para_id)) \
-    --ws-port $((9946 + $para_id)) \
     --rpc-external \
     --rpc-cors all \
-    --ws-external \
     --rpc-methods=Unsafe \
-    --state-cache-size 0 \
```

Comment:
**NIT**

We might wanna consider a `trie-cache size` value, but not sure if needed.

Reply:
The trie cache size is set by default. Should be fine for now.